internal/db/sqltest/tests/alias/wt_target_alias.sql

```diff
@@ -0,0 +1,141 @@
+-- Copyright (c) HashiCorp, Inc.
+-- SPDX-License-Identifier: BUSL-1.1
+
+-- controller_id tests:
+--  validates the wt_controller_id domain
+
+begin;
+  select plan(17);
+
+  select has_domain('wt_target_alias');
+  
+  create table target_alias_testing (
+    v wt_target_alias
+  );
+
+  prepare empty_insert as insert into target_alias_testing (v) values ('');
+  select throws_like(
+    'empty_insert',
+    '%"wt_target_alias_too_short"',
+    'We should error for empty values'
+  );
+  
+  prepare null_insert as insert into target_alias_testing (v) values (null);
+  select throws_like(
+    'null_insert',
+    'domain wt_target_alias does not allow null values'
+  );
+
+  prepare valid_inserts as insert into target_alias_testing (v) values 
+    ('a'),
+    ('A'),
+    ('192.168.1.9-9'),
+    ('foo'),
+    ('a.b.c'),
+    ('A.B.C'),
+    ('a-b-c'),
+    ('a.9-9'),
+    ('9things'),
+    ('9-things'),
+    ('hp--something.test.com'),
+    ('test-for-long-name-which-is-almost-over-the-limit-of-characters'),
+    ('TEST-FOR-LONG-NAME-WHICH-IS-ALMOST-OVER-THE-LIMIT-OF-CHARACTERS'),
+    ('test-for-long-name-which-is-almost-over-the-limit-of-characters.another-label'),
+    ('test.test-for-long-name-which-is-almost-over-the-limit-of-characters'),
+    ('test-for-long-name-which-is-almost-over-the-limit-of-characters.test-for-long-name-which-is-almost-over-the-limit-of-characters'),
+    ('9-things.9-things');
+  select lives_ok('valid_inserts');
+
+  prepare label_to_long as insert into target_alias_testing (v) values
```

Comment:
Nit: to -> too here and in the tests below
```suggestion
  prepare label_too_long as insert into target_alias_testing (v) values
```